<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="states">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>541</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="defaultString">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>String</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valid chars</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="validChars">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberStates">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of states</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoComplete">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Complete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextAction">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newMachine">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Жми, екарный бабай!</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="showMeWhatYouGot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1281</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнил</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTuring_machine_go_br_r_r_r">
    <property name="title">
     <string>Turing machine go br-r-r-r</string>
    </property>
   </widget>
   <addaction name="menuTuring_machine_go_br_r_r_r"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
